package.nls.it.json

```diff
@@ -43,31 +43,31 @@
   "configuration.SAS.connectionProfiles.profiles.context": "Contesto di SAS Viya",
   "configuration.SAS.connectionProfiles.profiles.endpoint": "Endpoint del profilo di connessione SAS Viya",
   "configuration.SAS.connectionProfiles.profiles.iom.host": "Host di Connessione SAS IOM",
-  "configuration.SAS.connectionProfiles.profiles.iom.port": "porta di Connessione SAS IOM",
-  "configuration.SAS.connectionProfiles.profiles.iom.username": "Username per Connessione SAS IOM",
+  "configuration.SAS.connectionProfiles.profiles.iom.port": "Porta di Connessione SAS IOM",
+  "configuration.SAS.connectionProfiles.profiles.iom.username": "Nome utente per connessione SAS IOM",
   "configuration.SAS.connectionProfiles.profiles.name": "Nome del profilo di connessione SAS",
   "configuration.SAS.connectionProfiles.profiles.sasOptions": "Opzioni della connessione SAS",
   "configuration.SAS.connectionProfiles.profiles.ssh.host": "Host della connessione SAS SSH",
   "configuration.SAS.connectionProfiles.profiles.ssh.port": "Porta della connessione SAS SSH",
   "configuration.SAS.connectionProfiles.profiles.ssh.privateKeyFilePath": "Percorso del file della chiave privata della connessione SSH SAS",
   "configuration.SAS.connectionProfiles.profiles.ssh.saspath": "Percorso dell'eseguibile della connessione SAS SSH",
   "configuration.SAS.connectionProfiles.profiles.ssh.username": "Nome utente della connessione SAS SSH",
-  "configuration.SAS.flowConversionMode": "Scegli la modalità di conversione per i notebook",
+  "configuration.SAS.flowConversionMode": "Scegli la modalità  di conversione per i notebook",
```

Comment:
it looks like an unnecessary extra space though I'm not an expert on the language.